ng2-components/ng2-activiti-tasklist/src/components/activiti-people.component.spec.ts

```diff
@@ -148,36 +119,39 @@ describe('ActivitiPeople', () => {
         });
 
         it('should show people involved', () => {
-            expect(element.querySelector('#user-fake-id')).not.toBeNull();
-            expect(element.querySelector('#user-fake-id').textContent).toContain('fake-name');
-            expect(element.querySelector('#user-fake-id').textContent).toContain('fake-last');
+            fixture.whenStable()
+                .then(() => {
+                    fixture.detectChanges();
+                    expect(element.querySelector('.assignment-list-container')).not.toBeNull();
+                    expect(fixture.debugElement.queryAll(By.css('activiti-people-list alfresco-datatable tbody tr')).length).toBe(2);
+                });
         });
 
-        it('should remove pepole involved', async(() => {
```

Comment:
i don't think the test is testing correctly the behaviour without the async clause here.